README.md

```diff
@@ -83,6 +83,30 @@ The .app file appears in the dist directory and can be dragged to
 "My Applications". Please note that actual py2app buils do not work yet. This
 is an aliased build, so it will break if you delete your git clone.
 
+### Windows (experimental)
+Assuming you have no previous python installation, either 
+    * use a package manager like chocolatey (http://chocolatey.org/) to install python3
+    * or install Python 3.6 manually from https://www.python.org/downloads/windows/
+
+In the installer, make sure that 'Include python in PATH' or similar is checked. Then after the installation, start a new command prompt and type `python -V` or `pip3` to check if it was properly installed. If the version number and the help message are returned respectively, you are good to go further. If not, in case you already had `cmd.exe` open, restart it or execute `refreshenv` to reload environment variables.
+If it still does not work yet, check if the PATH was set in the GUI or manually with `cmd.exe`.
+
+** In case pip is found but does not seem to work (error such as: "Fatal error in launcher: Unable to create process using `"`) you can start pip with `python -m pip` and then give the arguments like normal. **
```

Comment:
Could you explain this line? I can't imagine pip giving an error like this?

Reply:
Also, this section is rather long for something that people probably won't run into. How do you feel about moving this and the PATH GUI and CMD instructions below to the Troubleshooting section a bit below?

Reply:
Sure we can move it below. This error I ran into. While I didn't remember it from the past, when searching for it it seemed very common, is described here https://github.com/pypa/pip/issues/1997 among others. But sure I can remove it if you prefer.

Reply:
Reading another comment, I think I know why it happened for me. I also had a Python 2 install that I didn't want to remove, so I had to rename the Python 3 executable to python3.exe. Someone said that this breaks the 'launcher'.exe that python uses on Windows. I cannot test this right now, but I guess then I can remove that part.

Reply:
Ah, okay. The error looked suspiciously much like an error in pext_module_launcher so I got confused for a bit. I understand now and you can keep the line (it's actually helpful as a real error message), thanks!

Reply:
Moved.